commit 91de550243121056984e5b9b693b486860655d31
Author: Sergei Trofimovich <slyich@gmail.com>
Date:   Sat Dec 28 19:11:19 2024 +0000

    libcamera: Add missing <stdint.h> include to dma_buf_allocator.h
    
    Without the change the build fails on upcoming `gcc-15` as:
    
        In file included from ../src/libcamera/dma_buf_allocator.cpp:9:
        ../include/libcamera/internal/dma_buf_allocator.h:66:19: error: 'uint64_t' has not been declared
           66 |         void sync(uint64_t step);
              |                   ^~~~~~~~
    
    Signed-off-by: Sergei Trofimovich <slyich@gmail.com>
    Reviewed-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
    Reviewed-by: Kieran Bingham <kieran.bingham@ideasonboard.com>
    Signed-off-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>

diff --git a/include/libcamera/internal/dma_buf_allocator.h b/include/libcamera/internal/dma_buf_allocator.h
index d26f8a74..13600915 100644
--- a/include/libcamera/internal/dma_buf_allocator.h
+++ b/include/libcamera/internal/dma_buf_allocator.h
@@ -8,6 +8,7 @@
 #pragma once
 
 #include <memory>
+#include <stdint.h>
 #include <string>
 #include <vector>
 
